lib/route/route.js

```diff
@@ -10,6 +10,10 @@ const cloneDeep = require('lodash.clonedeep')
 const { FQD, PORT } = require('~/fb-runner-node/constants/constants')
 const SERVER_ADDRESS = FQD ? FQD.replace(/\/$/, '') : `http://localhost:${PORT}`
 
+const isUploadCheckPage = ({ _type } = {}) => _type === 'page.uploadCheck'
```

Comment:
We currently have no `common` in **Runner** but these functions are being duplicated all over

I plan to clean them up when `upload` is released (if not sooner)